spack-environments/csd3/spack.yaml

```diff
@@ -101,9 +101,51 @@ spack:
       modules: []
       environment: {}
       extra_rpaths: []
+  - compiler:
+      spec: intel@18.0.5
+      paths:
+        cc: /usr/local/Cluster-Apps/intel/2018.4/compilers_and_libraries_2018.5.274/linux/bin/intel64/icc
+        cxx: /usr/local/Cluster-Apps/intel/2018.4/compilers_and_libraries_2018.5.274/linux/bin/intel64/icpc
+        f77: /usr/local/Cluster-Apps/intel/2018.4/compilers_and_libraries_2018.5.274/linux/bin/intel64/ifort
+        fc: /usr/local/Cluster-Apps/intel/2018.4/compilers_and_libraries_2018.5.274/linux/bin/intel64/ifort
+      flags: {}
+      operating_system: scientific7
+      target: x86_64
+      modules: []
+      environment: {}
+      extra_rpaths: []
+  - compiler:
+      spec: intel@19.0.4.243
+      paths:
+        cc: /usr/local/Cluster-Apps/intel/2019.4/compilers_and_libraries_2019.4.243/linux/bin/intel64/icc
+        cxx: /usr/local/Cluster-Apps/intel/2019.4/compilers_and_libraries_2019.4.243/linux/bin/intel64/icpc
+        f77: /usr/local/Cluster-Apps/intel/2019.4/compilers_and_libraries_2019.4.243/linux/bin/intel64/ifort
+        fc: /usr/local/Cluster-Apps/intel/2019.4/compilers_and_libraries_2019.4.243/linux/bin/intel64/ifort
+      flags: {}
+      operating_system: scientific7
+      target: x86_64
+      modules: []
+      environment: {}
+      extra_rpaths: []
+  - compiler:
+      spec: intel@19.1.2.254
+      paths:
+        cc: /usr/local/Cluster-Apps/intel/2020.2/compilers_and_libraries_2020.2.254/linux/bin/intel64/icc
+        cxx: /usr/local/Cluster-Apps/intel/2020.2/compilers_and_libraries_2020.2.254/linux/bin/intel64/icpc
+        f77: /usr/local/Cluster-Apps/intel/2020.2/compilers_and_libraries_2020.2.254/linux/bin/intel64/ifort
+        fc: /usr/local/Cluster-Apps/intel/2020.2/compilers_and_libraries_2020.2.254/linux/bin/intel64/ifort
+      flags: {}
+      operating_system: scientific7
+      target: x86_64
+      modules: []
+      environment: {}
+      extra_rpaths: []
   packages:
     openmpi:
       externals:
+      - spec: openmpi@4.0.0~cuda+cxx~cxx_exceptions~java~memchecker+pmi+pmix~sqlite3+static~thread_multiple~wrapper-rpath
```

Comment:
If you rebase on `main` you'll get also `intel-mpi` from the system, even though that's probably not going to be used anyways unless you force this MPI provider in the Spack spec with `^intel-mpi`

Reply:
I don't have strong feelings over whether I should request `intel-mpi` or not. I guess ideally that should be a `parameter()`

Reply:
It is now a parameter.